sql/char_fishing.sql

```diff
@@ -0,0 +1,22 @@
+-- --------------------------------------------------------
+-- Add history for player fishing data
+-- --------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS `char_fishing` (
+    -- Fishing Catch History
+    `charid`           int(10) unsigned NOT NULL,
+    `fish_list`        blob,
+    `fish_linescast`   int(6)  unsigned NOT NULL DEFAULT 0,
+    `fish_reeled`      int(5)  unsigned NOT NULL DEFAULT 0,
+    `fish_longest`     int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_longest_id`  int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_heaviest`    int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_heaviest_id` int(4)  unsigned NOT NULL DEFAULT 0,
+
+    -- Fishing Contest History
+    `contest_rank_1`   smallint(3)  unsigned NOT NULL DEFAULT 0,
+    `contest_rank_2`   smallint(3)  unsigned NOT NULL DEFAULT 0,
+    `contest_rank_3`   smallint(3)  unsigned NOT NULL DEFAULT 0,
+    `contest_rank_4`   smallint(3)  unsigned NOT NULL DEFAULT 0,
```

Comment:
Why are catch and contest history split here, but are part of the same table? Is this hinting that they should be separate tables?

Reply:
No reason they can't be.  I only grouped them because they're both history of the character's fishing activities.  They can be char_fishing_history and char_fishing_contest_history if that's preferred.